app/views/tournaments/_deck_selection.html.slim

```diff
@@ -0,0 +1,25 @@
+.col-md-6
+  .card
+    .card-header
+      | Please select from your decks below.
+      a< href="https://netrunnerdb.com/en/decks" target="_blank" Set them in NetrunnerDB.
```

Comment:
I am not sure what “Set them in NetrunnerDB means here.”  I appreciate a link to the my decks page, but would pick something like “See your decks on NetrunnerDB.”.  How does one update if changes were made in NRDB?  Just a hard reload?

Reply:
Yes, you'd have to refresh the screen. Thanks very much, that does sound clearer. I might add a bit more.

Reply:
I've updated the text.